projects/cpython3/gen_fuzz.py

```diff
@@ -0,0 +1,47 @@
+"""Generate a fuzz test.
```

Comment:
> shouldn't this file be part of the python trunk?

I expect the code in this script to eventually be shared across both cpython2/cpython3, and probably also non-cpython fuzz tests that use the python runtime (e.g. simplejson). It could go in cpython, it just seems like there'll be slightly more duplication if I do. (Also, it'd be "dead code" in cpython's repo as it would never be executed).

Does that make sense?

> Also, do you really need to write this in python?
> Even a simple C include will work

I guess I am terrified of `#include SOME_CONSTANT`. Does that work if I do `-D 'SOME_CONSTANT="path/to/fuzz_foo.inc"'`? Or are you thinking I should use `SOME_CONSTANT` and pass `-D "source=$(cat path/to/fuzz_foo.inc)"`?